CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 cmake_minimum_required(VERSION 3.14) # for add_link_options and implicit target directories.
 project("llama.cpp" C CXX)
+
+
+if(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+    add_compile_options(-mfma -mavx2)
+endif()
+
```

Comment:
You should remove that, as we already support dynamic backend loading based on detected CPU features at runtime.